docs/AddingAMapLayer.md

```diff
@@ -0,0 +1,38 @@
+# Adding a map layer
+
+## map server
+
+Before you can draw a map layer in the digitransit UI you will have to add it to `hsl-map-server` first.
+
+An example of the layer source would be https://github.com/mfdz/tilelive-roadworks-bw
+
+This layer takes a GeoJSON source and then converts it to the [Mapbox vector tile format](https://github.com/mapbox/vt-pbf).
+This is a highly optimized format for sending geographic data to the client.
+
+Take a note the the [`id` you give to your layer](https://github.com/mfdz/tilelive-roadworks-bw/blob/master/index.js#L85).
```

Comment:
Typo. And how about adding a small code example directly to this page?

Reply:
Do you mean the source code of the layer? I'm not sure if a "small snippet" would contain the needed information but I can add it.